spec/features/proposals_spec.rb

```diff
@@ -203,6 +203,89 @@
         expect(page).not_to have_link("No comments", href: "#comments")
       end
     end
+
+    scenario "Shows clickable markers for related proposals by tag on proposal map", :js do
+      Setting["feature.map"] = true
+      create(:tag, :category, name: "culture")
+      create(:tag, :category, name: "sports")
```

Comment:
I think these two lines are not necessary; the `tag_list=` method automatically creates tags if they don't exist.